packages/recipe/src/components/EzWell/EzWell.tsx

```diff
@@ -36,16 +24,24 @@ function Well(props: Props, ref: Ref<HTMLDivElement>) {
     console.warn('A labelled Well must have a role.');
 
   return (
-    <div
+    <Stack
       {...filterDOMProps(otherProps, {
         propNames: new Set(['id', 'className']),
       })}
       role={role}
       ref={ref}
-      className={styles()}
+      p="20px"
+      sx={{
+        bgcolor: 'rgba(75, 75, 75, 0.02)',
+        border: '1px solid rgba(44,44,44,0.05)',
```

Comment:
This can be formatted with spaces for better readability.
```suggestion
        border: '1px solid rgba(44, 44, 44, 0.05)',
```